assignments/lambda-classes.js

```diff
@@ -1 +1,178 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor(attributes) {
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
+    }
+
+    speak() {
+        return `Hello my name is ${this.name}, I am from ${this.location}`;
+}
+}
+
+
+
+class Instructor extends Person {
+    constructor(instructorAttributes) {
```

Comment:
```suggestion
    constructor(props) {
```
I recommend just using props as the constructor/super variable. Technically if you think about it you're always just extending the "properties" of the parent's class. Plus, this will prepare you for the patterns used in React.